automation/check-patch.sh

```diff
@@ -103,7 +103,7 @@ main() {
     local run_path="$(get_run_path "$cluster")"
     local args=("prefix=$run_path")
     local inventory_file="$(realpath inventory)"
-    local storage_role="${STORAGE_ROLE:-storage-none}"
+    local storage_role="${STORAGE_ROLE:-storage-glusterfs}"
```

Comment:
Do not make this change. We are still waiting for https://github.com/kubevirt/kubevirt-ansible/pull/189 to be resolved so that we are working on any fixes in one place.